### ISSUE TITLE
Error: Fody: Could not find a weaver named 'BasicFodyAddin'

### ISSUE THREAD

**Issue by mfkl** [Fody/BasicFodyAddin#10]
* [x] Uploaded a minimal solution that reproduces the problem with the fewest moving pieces. Preferably a public GitHub repository.

Modified a tiny bit the original repository https://github.com/mfkl/BasicFodyAddin
Built it with `dotnet build -c Release BasicFodyAddin.sln`
Added it to the target project targeting netstandard2.0, xamarin.ios etc. as a `PackageReference`
Added a `PackageReference` to fody and a `FodyWeavers.xml` to the target project with
```
<?xml version="1.0" encoding="utf-8"?>
<Weavers>
    <BasicFodyAddin />
</Weavers>
``` 
No matter what I try, Fody does not find it. BasicFodyAddin does not end up in `SolutionDir/packages either`, though it is in `/Users/Me/.nuget/packages`. I can't figure out the part I'm missing to make this work.

**Comment by SimonCropp** [comment]
Does it worl with un-modified weavers?

**Comment by mfkl** [comment]
No.

I'm on Mac, and to be able to build this repo after a fresh clone I had to remove the `net4x` target frameworks to make a netstandard only plugin (that will operate on a `xamarin.ios` assembly later).

I managed to make Fody find the plugin by renaming the folder "netstandardweaver" to "netclassicweaver". 

I haven't tried building on Windows, but I suspect I'd not encounter this issue.

Are you sure of these lines?
https://github.com/Fody/Fody/blob/d4ecec3fb87c5e770abeb9bd223478b6494747cd/Fody/AddinFinder/AddinSearchDirectories.cs#L116
